docs/workforce-management-framework/topics/con-release-notes.adoc

```diff
@@ -3,22 +3,22 @@
 == Overview
 
 {PROJECT_NAME} allows you to move from paper based forms and processes into complete mobile solution.
-{PROJECT_NAME} connects a business back-office portal application with its fleet of mobilized employees via a mobile appliction. 
-Both mobile and portal applciations are fully customizable.
+{PROJECT_NAME} connects a business back-office portal application with its fleet of mobilized employees via a mobile application. 
+Both mobile and portal applications are fully customizable.
 
 == About This Release
-{PROJECT_NAME} {WFM-RC-Version} is a new produce was previosly in tech preview. The Features are listed below.
+{PROJECT_NAME} {WFM-RC-Version} is a new produce was previously in tech preview. The Features are listed below.
```

Comment:
> {PROJECT_NAME} {WFM-RC-Version} is a new produce was previously in tech preview

this sentence doesn't really make sense.
How about
{PROJECT_NAME} is now out of tech preview and release {WFM-RC-Version} has following features:
?